In [1]:
import pandas as pd
import numpy as np

In [2]:
import os
print(os.getcwd())

/Users/eunbin143/Library/CloudStorage/GoogleDrive-silverbbinni@gmail.com/내 드라이브/빅데이터-프로젝트


In [3]:
%cd '/Users/eunbin143/Library/CloudStorage/GoogleDrive-silverbbinni@gmail.com/내 드라이브/빅데이터-프로젝트/dataFolder/'

/Users/eunbin143/Library/CloudStorage/GoogleDrive-silverbbinni@gmail.com/내 드라이브/빅데이터-프로젝트/dataFolder


In [4]:
data = pd.read_csv('cqrs2016.csv', encoding='iso-8859-1', sep=';')
data.head()

za_nr             version                  doi           woche      lfd  \
0   6628  1.0.0 (2017-07-11)  doi:10.4232/1.12826  KW 03 - Januar  2991579   
1   6628  1.0.0 (2017-07-11)  doi:10.4232/1.12826  KW 03 - Januar  3192308   
2   6628  1.0.0 (2017-07-11)  doi:10.4232/1.12826  KW 03 - Januar  3219321   
3   6628  1.0.0 (2017-07-11)  doi:10.4232/1.12826  KW 03 - Januar  3236576   
4   6628  1.0.0 (2017-07-11)  doi:10.4232/1.12826  KW 03 - Januar  3260401   

    gewicht       datum                bik                   bl    wo  ...  \
0  1,388719  2016/01/13  100000 bis 500000    Baden-Württemberg  West  ...   
1   ,596671  2016/01/13     5000 bis 20000  Nordrhein-Westfalen  West  ...   
2   ,266309  2016/01/13    500000 und mehr               Berlin   Ost  ...   
3   ,085094  2016/01/13    500000 und mehr               Berlin  West  ...   
4  1,429932  2016/01/13           bis 2000            Thüringen   Ost  ...   

     s8                   s9        s10                      V1 V1b  \
0    ja  2500 bis unter 3000  Mobilfunk  mache mir keine Sorgen       
1    ja  1500 bis unter 2000  Mobilfunk        mache mir Sorgen       
2  nein         keine Angabe  Mobilfunk        mache mir Sorgen       
3  nein   500 bis unter 1000  Mobilfunk  mache mir keine Sorgen       
4    ja  1000 bis unter 1500  Mobilfunk  mache mir keine Sorgen       

                                                V2_1  \
0  oder gleichen sich Vor- und Nachteile in etwa ...   
1                                     eher Nachteile   
2  oder gleichen sich Vor- und Nachteile in etwa ...   
3                                     eher Vorteile,   
4  oder gleichen sich Vor- und Nachteile in etwa ...   

                                                V2_2                      V3  \
0                                     eher Vorteile,  kommt eher nicht voran   
1  oder gleichen sich Vor- und Nachteile in etwa ...  kommt eher nicht voran   
2                                     eher Nachteile  kommt eher nicht voran   
3                                     eher Vorteile,  kommt eher nicht voran   
4  oder gleichen sich Vor- und Nachteile in etwa ...        kommt eher voran   

  V3b V3c  
0          
1          
2          
3          
4          

[5 rows x 27 columns]

In [5]:
data.drop('za_nr', axis =1, inplace=True)
data.drop('version', axis =1, inplace=True)
data.drop('doi', axis =1, inplace=True)
data.drop('s10', axis =1, inplace=True)
data.head()

woche      lfd   gewicht       datum                bik  \
0  KW 03 - Januar  2991579  1,388719  2016/01/13  100000 bis 500000   
1  KW 03 - Januar  3192308   ,596671  2016/01/13     5000 bis 20000   
2  KW 03 - Januar  3219321   ,266309  2016/01/13    500000 und mehr   
3  KW 03 - Januar  3236576   ,085094  2016/01/13    500000 und mehr   
4  KW 03 - Januar  3260401  1,429932  2016/01/13           bis 2000   

                    bl    wo s1  s2                   s3  ...  \
0    Baden-Württemberg  West  m  46              Studium  ...   
1  Nordrhein-Westfalen  West  w  51               Abitur  ...   
2               Berlin   Ost  m  16  Volks-, Hauptschule  ...   
3               Berlin  West  m  74              Studium  ...   
4            Thüringen   Ost  m  57  Volks-, Hauptschule  ...   

                         s7    s8                   s9  \
0   Bündnis `90, Die Grünen    ja  2500 bis unter 3000   
1  weiß nicht, keine Angabe    ja  1500 bis unter 2000   
2                            nein         keine Angabe   
3                            nein   500 bis unter 1000   
4                 Die Linke    ja  1000 bis unter 1500   

                       V1 V1b  \
0  mache mir keine Sorgen       
1        mache mir Sorgen       
2        mache mir Sorgen       
3  mache mir keine Sorgen       
4  mache mir keine Sorgen       

                                                V2_1  \
0  oder gleichen sich Vor- und Nachteile in etwa ...   
1                                     eher Nachteile   
2  oder gleichen sich Vor- und Nachteile in etwa ...   
3                                     eher Vorteile,   
4  oder gleichen sich Vor- und Nachteile in etwa ...   

                                                V2_2                      V3  \
0                                     eher Vorteile,  kommt eher nicht voran   
1  oder gleichen sich Vor- und Nachteile in etwa ...  kommt eher nicht voran   
2                                     eher Nachteile  kommt eher nicht voran   
3                                     eher Vorteile,  kommt eher nicht voran   
4  oder gleichen sich Vor- und Nachteile in etwa ...        kommt eher voran   

  V3b V3c  
0          
1          
2          
3          
4          

[5 rows x 23 columns]

In [6]:
data.rename(columns={'datum': 'date'}, inplace=True)
data.rename(columns={'wo': 'West/East'}, inplace=True)
data.rename(columns={'s1': 'Gender'}, inplace=True)
data.rename(columns={'s2': 'Age'}, inplace=True)
data.rename(columns={'s3': 'Education'}, inplace=True)
data.rename(columns={'s4': 'Job'}, inplace=True)
data.rename(columns={'s5': 'Household size'}, inplace=True)
data.rename(columns={'s6': 'NumberofPerson>14'}, inplace=True)
data.rename(columns={'s7': 'party preference'}, inplace=True)
data.rename(columns={'s8': 'Eligible to vote'}, inplace=True)
data.rename(columns={'s9': 'HH Income'}, inplace=True)
data.rename(columns={'bl': 'Bundesland'}, inplace=True)
data.rename(columns={'bik': 'location_size'}, inplace=True)

In [7]:
data

woche       lfd   gewicht        date      location_size  \
0        KW 03 - Januar   2991579  1,388719  2016/01/13  100000 bis 500000   
1        KW 03 - Januar   3192308   ,596671  2016/01/13     5000 bis 20000   
2        KW 03 - Januar   3219321   ,266309  2016/01/13    500000 und mehr   
3        KW 03 - Januar   3236576   ,085094  2016/01/13    500000 und mehr   
4        KW 03 - Januar   3260401  1,429932  2016/01/13           bis 2000   
...                 ...       ...       ...         ...                ...   
13124  KW 50 - Dezember  34804053   ,498295  2016/12/13    20000 bis 50000   
13125  KW 50 - Dezember  34867180   ,489411  2016/12/13    20000 bis 50000   
13126  KW 50 - Dezember  34881488  1,050764  2016/12/13           bis 2000   
13127  KW 50 - Dezember  34932962   ,290195  2016/12/13     5000 bis 20000   
13128  KW 50 - Dezember  35030457   ,172611  2016/12/13           bis 2000   

                Bundesland West/East Gender  Age            Education  ...  \
0        Baden-Württemberg      West      m   46              Studium  ...   
1      Nordrhein-Westfalen      West      w   51               Abitur  ...   
2                   Berlin       Ost      m   16  Volks-, Hauptschule  ...   
3                   Berlin      West      m   74              Studium  ...   
4                Thüringen       Ost      m   57  Volks-, Hauptschule  ...   
...                    ...       ...    ...  ...                  ...  ...   
13124            Thüringen       Ost      w   58              Studium  ...   
13125            Thüringen       Ost      w   77              Studium  ...   
13126            Thüringen       Ost      w   79  Volks-, Hauptschule  ...   
13127            Thüringen       Ost      m   73              Studium  ...   
13128            Thüringen       Ost      m   79              Studium  ...   

               party preference Eligible to vote            HH Income  \
0       Bündnis `90, Die Grünen               ja  2500 bis unter 3000   
1      weiß nicht, keine Angabe               ja  1500 bis unter 2000   
2                                           nein         keine Angabe   
3                                           nein   500 bis unter 1000   
4                     Die Linke               ja  1000 bis unter 1500   
...                         ...              ...                  ...   
13124                   CDU/CSU               ja  2500 bis unter 3000   
13125  weiß nicht, keine Angabe               ja  1500 bis unter 2000   
13126                   CDU/CSU               ja         keine Angabe   
13127                 Die Linke               ja  2500 bis unter 3000   
13128                   CDU/CSU               ja  2500 bis unter 3000   

                           V1                     V1b  \
0      mache mir keine Sorgen                           
1            mache mir Sorgen                           
2            mache mir Sorgen                           
3      mache mir keine Sorgen                           
4      mache mir keine Sorgen                           
...                       ...                     ...   
13124                          mache mir keine Sorgen   
13125                          mache mir keine Sorgen   
13126                          mache mir keine Sorgen   
13127                                mache mir Sorgen   
13128                          mache mir keine Sorgen   

                                                    V2_1  \
0      oder gleichen sich Vor- und Nachteile in etwa ...   
1                                         eher Nachteile   
2      oder gleichen sich Vor- und Nachteile in etwa ...   
3                                         eher Vorteile,   
4      oder gleichen sich Vor- und Nachteile in etwa ...   
...                                                  ...   
13124  oder gleichen sich Vor- und Nachteile in etwa ...   
13125  oder gleichen sich Vor- und Nachteile in etwa ...   
13126                                     

V1 [Januar – September, Oktober (Split A)] Machen Sie sich Sorgen darüber, dass so viele Flüchtlinge nach Deutschland kommen, oder machenSie sich keine Sorgen? = 독일로 많은 난민들이 오는 것에 대하여 걱정하시나요? 아니면 걱정하지 않으시나요?

V1b [Oktober (Split B), November, Dezember]Machen Sie sich Sorgen darüber, dass so viele Flüchtlinge in Deutschland sind, oder machen Sie sichkeine Sorgen? = 독일에 이렇게 많은 난민들이 있는 것에 대하여 걱정하시나요? 아니면 걱정하지 않으시나요

답변) 
 1: mache mir Sorgen = 걱정합니다. 
 2: mache mir keine Sorgen = 걱정하지 않습니다. 
 9: weiß nicht, keine Angabe = 모르겠습니다. 

V1과 V1b는 같은 의미의 질문으로, 질문 시기의 차이로 인해 표현만 다를 뿐이므로, 하나의 통합된 열로 결합해야한다. 

In [8]:
value_counts_v1=data['V1'].value_counts()
print(value_counts_v1)
value_counts_v1b=data['V1b'].value_counts()
print(value_counts_v1b)

V1
mache mir Sorgen            5287
mache mir keine Sorgen      5051
                            2522
weiß nicht, keine Angabe     269
Name: count, dtype: int64
V1b
                            10607
mache mir keine Sorgen       1422
mache mir Sorgen             1064
weiß nicht, keine Angabe       36
Name: count, dtype: int64


- V1의 데이터 개수 (5287+5051+269) = 10607

- 공백값 = 2522
- V1b 의 공백값 = 10607
- => 중복되는 값이 없기에 numpy where을 사용하여 결합해도 데이터를 손상시키지 않을 것 
- V1 = '난민에 관한 우려' 로 제목 변경

In [9]:
data['난민에 관한 우려'] = data['V1'].fillna(data['V1b'])
data.drop('V1', axis =1, inplace=True)
data.drop('V1b', axis =1, inplace=True)

In [10]:
data.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 13129 entries, 0 to 13128
Data columns (total 22 columns):
 #   Column             Non-Null Count  Dtype 
---  ------             --------------  ----- 
 0   woche              13129 non-null  object
 1   lfd                13129 non-null  int64 
 2   gewicht            13129 non-null  object
 3   date               13129 non-null  object
 4   location_size      13129 non-null  object
 5   Bundesland         13129 non-null  object
 6   West/East          13129 non-null  object
 7   Gender             13129 non-null  object
 8   Age                13129 non-null  int64 
 9   Education          13129 non-null  object
 10  Job                13129 non-null  object
 11  Household size     13129 non-null  object
 12  NumberofPerson>14  13129 non-null  object
 13  party preference   13129 non-null  object
 14  Eligible to vote   13129 non-null  object
 15  HH Income          13129 non-null  object
 16  V2_1               13129 non-null  objec

In [11]:
#이전 위치로 옮기기
cols = data.columns.tolist()
cols.insert(16, cols.pop(cols.index('난민에 관한 우려')))
data = data[cols]

V2: Hat die Aufnahme von Flüchtlingen für Deutschland eher Vorteile, eher Nachteile oder gleichen sich Vor- und Nachteile in etwa aus?
= 난민의 수용이 독일에 단기적/장기적으로 이익이 될지 손해가 될지

    V2_1 - kurzfristig 단기적 관점에서 난민 수용의 영향에 대한 인식

    V2_2 - langfristig 장기적 관점에서 난민 수용의 영향에 대한 인식

답변) 
 1: eher Vorteile = 오히려 장점이 많다.
 2: eher Nachteilie = 오히려 단점이 많다.
 9: weiß nicht, keine Angabe = 모르겠습니다. 

In [12]:
value_counts_v2_1=data['V2_1'].value_counts()
print(value_counts_v2_1)
value_counts_v2_2=data['V2_2'].value_counts()
print(value_counts_v2_2)

V2_1
eher Nachteile                                        7258
oder gleichen sich Vor- und Nachteile in etwa aus?    4299
eher Vorteile,                                         875
weiß nicht, keine Angabe                               697
Name: count, dtype: int64
V2_2
oder gleichen sich Vor- und Nachteile in etwa aus?    4756
eher Nachteile                                        4143
eher Vorteile,                                        3372
weiß nicht, keine Angabe                               858
Name: count, dtype: int64


In [13]:
print(value_counts_v2_1.sum())
print(value_counts_v2_2.sum())

13129
13129


In [14]:
data.rename(columns={'V2_1': '단기적 난민 수용 인식'}, inplace=True)
data.rename(columns={'V2_2': '장기적 난민 수용 인식'}, inplace=True)

<h2> V3 [Januar – September, Oktober (Split C), November (Split A)]<br/></h2>
<div>    
Und wenn es um die Bewältigung der Flüchtlingskrise geht: Was meinen Sie, kommt die Bundesregierung da eher voran oder eher nicht voran?
= “난민 위기 극복과 관련하여, 연방정부가 이 문제를 잘 해결하고 있다고 생각하십니까, 아니면 그렇지 않다고 생각하십니까?”
</div>
<div>
<br/>
V3b [Oktober (Split D)]
Und wenn es um die Integration der Flüchtlinge geht: Was meinen Sie, kommt die Bundesregierung da eher voran oder eher nicht voran?
= “난민 통합과 관련하여, 연방정부가 이 문제를 잘 해결하고 있다고 생각하십니까, 아니면 그렇지 않다고 생각하십니까?”<br/>
</div>

<div>
<br/>
V3c [November (Split B), Dezember]
Und wenn es um die Bewältigung der Flüchtlingssituation geht: Was meinen Sie, kommt die Bundesregierung da eher voran oder eher nicht voran?
= “난민 상황의 극복과 관련하여, 연방정부가 이 문제를 잘 해결하고 있다고 생각하십니까, 아니면 그렇지 않다고 생각하십니까?”<br/>
</div>
<div>
    
<br/>
답변) 
<li> 1: kommt eher voran = 잘 해결하고 있다. </li>
<li> 2: kommt eher nicht voran = 잘 해결하지 못하고 있다. </li>
<li> 9: weiß nicht, keine Angabe = 모르겠습니다.</li>

모두, 다 같은 질문으로 통합할 수 있다. 
</div>

In [15]:
#v3의 값에 공백을 넣어둬서, 값으로 처리가 되어서 오류 발생했었음.. 전처리 확실히 하기
data['V3'] = data['V3'].replace(r'^\s*$', pd.NA, regex=True)
data['V3b'] = data['V3b'].replace(r'^\s*$', pd.NA, regex=True)
data['V3c'] = data['V3c'].replace(r'^\s*$', pd.NA, regex=True)

In [16]:
value_counts_v3_a=data['V3'].value_counts()
print(value_counts_v3_a)
print(sum(value_counts_v3_a))
print('\n')
value_counts_v3_b=data['V3b'].value_counts()
print(value_counts_v3_b)
print(sum(value_counts_v3_b))
print('\n')
value_counts_v3_c=data['V3c'].value_counts()
print(value_counts_v3_c)
print(sum(value_counts_v3_c))

V3
kommt eher nicht voran      7272
kommt eher voran            3226
weiß nicht, keine Angabe     620
Name: count, dtype: int64
11118


V3b
kommt eher nicht voran      318
kommt eher voran            137
weiß nicht, keine Angabe     31
Name: count, dtype: int64
486


V3c
kommt eher nicht voran      929
kommt eher voran            536
weiß nicht, keine Angabe     60
Name: count, dtype: int64
1525


In [17]:
data[['V3', 'V3b', 'V3c']].isnull().sum()

V3      2011
V3b    12643
V3c    11604
dtype: int64

In [18]:
# V3, V3b, V3c 중 하나의 값이 있으면 그 값을 사용
data['난민 문제에 대한 정부 인식'] = data['V3c'].fillna(data['V3b'])

# 기존 열 삭제
data.drop(columns=['V3b', 'V3c'], inplace=True)

data

woche       lfd   gewicht        date      location_size  \
0        KW 03 - Januar   2991579  1,388719  2016/01/13  100000 bis 500000   
1        KW 03 - Januar   3192308   ,596671  2016/01/13     5000 bis 20000   
2        KW 03 - Januar   3219321   ,266309  2016/01/13    500000 und mehr   
3        KW 03 - Januar   3236576   ,085094  2016/01/13    500000 und mehr   
4        KW 03 - Januar   3260401  1,429932  2016/01/13           bis 2000   
...                 ...       ...       ...         ...                ...   
13124  KW 50 - Dezember  34804053   ,498295  2016/12/13    20000 bis 50000   
13125  KW 50 - Dezember  34867180   ,489411  2016/12/13    20000 bis 50000   
13126  KW 50 - Dezember  34881488  1,050764  2016/12/13           bis 2000   
13127  KW 50 - Dezember  34932962   ,290195  2016/12/13     5000 bis 20000   
13128  KW 50 - Dezember  35030457   ,172611  2016/12/13           bis 2000   

                Bundesland West/East Gender  Age            Education  ...  \
0        Baden-Württemberg      West      m   46              Studium  ...   
1      Nordrhein-Westfalen      West      w   51               Abitur  ...   
2                   Berlin       Ost      m   16  Volks-, Hauptschule  ...   
3                   Berlin      West      m   74              Studium  ...   
4                Thüringen       Ost      m   57  Volks-, Hauptschule  ...   
...                    ...       ...    ...  ...                  ...  ...   
13124            Thüringen       Ost      w   58              Studium  ...   
13125            Thüringen       Ost      w   77              Studium  ...   
13126            Thüringen       Ost      w   79  Volks-, Hauptschule  ...   
13127            Thüringen       Ost      m   73              Studium  ...   
13128            Thüringen       Ost      m   79              Studium  ...   

      Household size NumberofPerson>14          party preference  \
0               zwei              zwei   Bündnis `90, Die Grünen   
1               drei              drei  weiß nicht, keine Angabe   
2        eine Person       eine Person                             
3        eine Person       eine Person                             
4               zwei              zwei                 Die Linke   
...              ...               ...                       ...   
13124           zwei              zwei                   CDU/CSU   
13125           zwei              zwei  weiß nicht, keine Angabe   
13126           zwei              zwei                   CDU/CSU   
13127           zwei              zwei                 Die Linke   
13128           zwei              zwei                   CDU/CSU   

      Eligible to vote            HH Income               난민에 관한 우려  \
0                   ja  2500 bis unter 3000  mache mir keine Sorgen   
1                   ja  1500 bis unter 2000        mache mir Sorgen   
2                 nein         keine Angabe        mache mir Sorgen   
3                 nein   500 bis unter 1000  mache mir keine Sorgen   
4                   ja  1000 bis unter 1500  mache mir keine Sorgen   
...                ...                  ...                     ...   
13124               ja  2500 bis unter 3000                           
13125               ja  1500 bis unter 2000                           
13126               ja         keine Angabe                           
13127               ja  2500 bis unter 3000                           
13128               ja  2500 bis unter 3000                           

                                            단기적 난민 수용 인식  \
0      oder gleichen sich Vor- und Nachteile in etwa ...   
1                                         eher Nachteile   
2      oder gleichen sich Vor- und Nachteile in etwa ...   
3                                         eher Vorteile,   
4      oder gleichen sich Vor- und Nachteile in etwa ...   
...                                                  ...   
13124  oder gleichen sich Vor- und Nachteile in etwa .

In [19]:
# V3, V3b, V3c 중 하나의 값이 있으면 그 값을 사용
data['난민문제에 대한 정부인식'] = data['난민 문제에 대한 정부 인식'].fillna(data['V3'])

# 기존 열 삭제
data.drop(columns=['V3'], inplace=True)
data.drop(columns=['난민 문제에 대한 정부 인식'], inplace=True)

data

woche       lfd   gewicht        date      location_size  \
0        KW 03 - Januar   2991579  1,388719  2016/01/13  100000 bis 500000   
1        KW 03 - Januar   3192308   ,596671  2016/01/13     5000 bis 20000   
2        KW 03 - Januar   3219321   ,266309  2016/01/13    500000 und mehr   
3        KW 03 - Januar   3236576   ,085094  2016/01/13    500000 und mehr   
4        KW 03 - Januar   3260401  1,429932  2016/01/13           bis 2000   
...                 ...       ...       ...         ...                ...   
13124  KW 50 - Dezember  34804053   ,498295  2016/12/13    20000 bis 50000   
13125  KW 50 - Dezember  34867180   ,489411  2016/12/13    20000 bis 50000   
13126  KW 50 - Dezember  34881488  1,050764  2016/12/13           bis 2000   
13127  KW 50 - Dezember  34932962   ,290195  2016/12/13     5000 bis 20000   
13128  KW 50 - Dezember  35030457   ,172611  2016/12/13           bis 2000   

                Bundesland West/East Gender  Age            Education   Job  \
0        Baden-Württemberg      West      m   46              Studium    ja   
1      Nordrhein-Westfalen      West      w   51               Abitur  nein   
2                   Berlin       Ost      m   16  Volks-, Hauptschule  nein   
3                   Berlin      West      m   74              Studium    ja   
4                Thüringen       Ost      m   57  Volks-, Hauptschule    ja   
...                    ...       ...    ...  ...                  ...   ...   
13124            Thüringen       Ost      w   58              Studium    ja   
13125            Thüringen       Ost      w   77              Studium  nein   
13126            Thüringen       Ost      w   79  Volks-, Hauptschule  nein   
13127            Thüringen       Ost      m   73              Studium  nein   
13128            Thüringen       Ost      m   79              Studium  nein   

      Household size NumberofPerson>14          party preference  \
0               zwei              zwei   Bündnis `90, Die Grünen   
1               drei              drei  weiß nicht, keine Angabe   
2        eine Person       eine Person                             
3        eine Person       eine Person                             
4               zwei              zwei                 Die Linke   
...              ...               ...                       ...   
13124           zwei              zwei                   CDU/CSU   
13125           zwei              zwei  weiß nicht, keine Angabe   
13126           zwei              zwei                   CDU/CSU   
13127           zwei              zwei                 Die Linke   
13128           zwei              zwei                   CDU/CSU   

      Eligible to vote            HH Income               난민에 관한 우려  \
0                   ja  2500 bis unter 3000  mache mir keine Sorgen   
1                   ja  1500 bis unter 2000        mache mir Sorgen   
2                 nein         keine Angabe        mache mir Sorgen   
3                 nein   500 bis unter 1000  mache mir keine Sorgen   
4                   ja  1000 bis unter 1500  mache mir keine Sorgen   
...                ...                  ...                     ...   
13124               ja  2500 bis unter 3000                           
13125               ja  1500 bis unter 2000                           
13126               ja         keine Angabe                           
13127               ja  2500 bis unter 3000                           
13128               ja  2500 bis unter 3000                           

                                            단기적 난민 수용 인식  \
0      oder gleichen sich Vor- und Nachteile in etwa ...   
1                                         eher Nachteile   
2      oder gleichen sich Vor- und Nachteile in etwa ...   
3                                         eher Vorteile,   
4      oder gleichen sich Vor- und Nachteile in etwa ...   
...                                                  ...   
13124  oder gleichen sich Vor- und Nachtei

In [22]:
data.to_csv('/Users/eunbin143/Library/CloudStorage/GoogleDrive-silverbbinni@gmail.com/내 드라이브/빅데이터-프로젝트/전처리완료data/workout_cqrs2016.csv', index=False)